src/execution/fixtures/ExecuteAfm.fixtures.ts

```diff
@@ -174,15 +174,45 @@ const tooLargeResponse: Execution.IError = {
     name: 'Error 413',
     message: 'Response is too large',
     response: {
-        status: 413
+        status: 413,
+        body: {} as ReadableStream, // tslint:disable-line
```

Comment:
`body: null as ReadableStream | null,`
taken from `lib.es6.d.ts` :-)

`+` maybe add // TODO will be solved in RAIL-650